### ISSUE TITLE
[Bug]: AC0032 with record ref

### ISSUE THREAD

**Issue by TKapitan** [ALCops/Analyzers#420]
### Analyzer

ApplicationCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

AC0032

### Description

When the object contains ANY RecordRef.Modify/Insert/Delete/Get the whole rule should be disabled as it cannot assess if the permission is needed. I've applied the code action and it removed some places where the permissions were needed.

### AL code to reproduce

```al
    Permissions =
        tabledata Microsoft.Bank.Ledger."Bank Account Ledger Entry" = md,
        tabledata Microsoft.Finance.GeneralLedger.Ledger."G/L Entry" = md,
        tabledata Microsoft.Finance.VAT.Ledger."VAT Entry" = md,
        tabledata Microsoft.Inventory.Ledger."Item Ledger Entry" = md,
        tabledata Microsoft.Inventory.Ledger."Value Entry" = md,
        tabledata Microsoft.Purchases.Payables."Detailed Vendor Ledg. Entry" = md,
        tabledata Microsoft.Purchases.Payables."Vendor Ledger Entry" = md,
        tabledata Microsoft.Sales.Receivables."Cust. Ledger Entry" = md,
        tabledata Microsoft.Sales.Receivables."Detailed Cust. Ledg. Entry" = md;

    procedure DoModify(var RecordRefToModify: RecordRef; RunTrigger: Boolean)
    begin
        RecordRefToModify.Modify(RunTrigger);
    end;
```

**Comment by Arthurvdv** [comment]
I've published a new beta release, `v1.1.0-beta.2`, which should resolve this issue.

Could you verify for me if it's fixed on your end? Would be great to hear whether it's resolved or you're still running into problems.

If you're using the [ALCops VS Code extension](https://marketplace.visualstudio.com/items?itemName=Arthurvdv.alcops), you can switch to the beta channel in the extension settings. To force an update check, run the `ALCops: Check for Updates` command from the Command Palette.

**Comment by Arthurvdv** [comment]
This should now be resolved with the release of `v1.1.0` of ALCops.

When you have a chance, could you update and verify whether the issue is fixed?

If everything is working as expected, please go ahead and close the issue. If you still run into the problem please leave the issue open and share any additional context, code samples or reproduction steps that might help me investigate further.
